src/main/java/uk/gov/hmcts/reform/jobscheduler/config/SwaggerConfig.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.jobscheduler.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+import uk.gov.hmcts.reform.jobscheduler.Application;
+
+import java.util.Collections;
+
+import static springfox.documentation.builders.RequestHandlerSelectors.basePackage;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+
+    @Bean
+    public Docket api() {
+        return new Docket(DocumentationType.SWAGGER_2)
+            .useDefaultResponseMessages(false)
+            .select()
+            .apis(basePackage(Application.class.getPackage().getName() + ".controllers"))
+            .paths(PathSelectors.any())
+            .build()
+            .apiInfo(new ApiInfo(
+                "Job Scheduler API",
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                Collections.emptyList()
```

Comment:
Hm, is there no constructor to just use the first argument instead of enlisting all nulls?